docs/source/install/rhel6.rst

```diff
@@ -6,23 +6,34 @@ This guide provides step-by step instructions on installing StackStorm on a sing
 <https://github.com/StackStorm/st2-packages/blob/master/scripts/st2bootstrap-el6.sh>`_, codifies the
 instructions below.
 
-.. warning :: Currently BETA! Please try, use and report bugs on
-   `github.com/StackStorm/st2-packages <https://github.com/StackStorm/st2-packages/issues/new>`_.
-   Soon, package-based installation will be
-   the preferred path to installing StackStorm.
+.. warning :: Currently in BETA! Upgrades are being tested, but will be supprted only once packages graduate
+    from BETA, likely from 1.4 onwards. At that point, package-based installation will be
+    the preferred path to installing StackStorm.
 
-.. warning ::
+    Please try, use and report bugs on
+    `github.com/StackStorm/st2-packages <https://github.com/StackStorm/st2-packages/issues/new>`_.
 
-    RHEL 6 doesn't seem to ship with libffi-devel which is a dependency for |st2|.
-    If that is the case, find a version of libffi-devel compatible with libffi on the box.
-    For example,
+.. contents::
+
+Supported platforms
+--------------------
 
-    .. code :: bash
+We support RedHat 7 / CentOS 7 and continously testing on `Red Hat Enterprise Linux (RHEL) 7.2 (HVM) Amazon AWS AMI <https://aws.amazon.com/marketplace/pp/B019NS7T5I/ref=srh_res_product_title?ie=UTF8&sr=0-2&qid=1457037671547>`_
```

Comment:
Probably missing "are" before "continuously testing" (typo in "continuously", btw). Not sure, I'm not a grammar expert, but two different tenses in one part of the sentence looks kinda weird. :)